index.html

```diff
@@ -170,10 +167,9 @@ <h3 class="title">Tonic</h3>
               </div>
             </div>
           </div>
-          <div class="cardfour noqs">
-            <a href="#">
-              <img class="cardimage" src="image/cardfour.svg" alt="" />
-            </a>
+          <div class="cardfour noqs  reversed">
+            <div class="cardimg card4">             
+            </div>
             <div class="card-title">
```

Comment:
- [ ] For the contact section kindly consider giving the form more width, and reducing the width of the subheading paragraph below `contact me`
- [ ] Kindly also make sure to use the right background image for this section (contact me) so your design looks identical to Figma's design

|Your design|Figma design|
|-|-|
|![Screenshot_20220901_214746](https://user-images.githubusercontent.com/25227022/188010103-3dccc156-d9c6-4a13-9abb-a0a80cb0ff47.png)|![Screenshot_20220901_214800](https://user-images.githubusercontent.com/25227022/188010126-5466ec78-845d-46c7-adb0-084cd56371e6.png)|

Reply:
Good job following the advice of the previous reviewer